Doc/library/re.rst

```diff
@@ -201,6 +201,18 @@ The special characters are:
      place it at the beginning of the set.  For example, both ``[()[\]{}]`` and
      ``[]()[{}]`` will both match a parenthesis.
 
+   * The support of nested sets and set operations can be added in future.
```

Comment:
Change to something like: "Support of nested sets and set operations as in Unicode ...[no idea what their equivalent name is for their 'peps'] 18 (link) might be added in the future. This would change the syntax, so to facilitate this change a  :exc:`FutureWarning` will be raised in ambiguous cases for the time being."